flink-streaming-connectors/pom.xml

```diff
@@ -45,6 +45,7 @@ under the License.
 		<module>flink-connector-rabbitmq</module>
 		<module>flink-connector-twitter</module>
 		<module>flink-connector-nifi</module>
+		<module>flink-connector-kinesis</module>
```

Comment:
I think we have to remove this line again. The module is included in the profile below (you have to activate the "include-kinesis" maven build profile)